chainstate/src/detail/mod.rs

```diff
@@ -44,15 +50,14 @@ use utxo::UtxosDBMut;
 use self::{
     orphan_blocks::{OrphanBlocksRef, OrphanBlocksRefMut},
     query::ChainstateQuery,
-    time_getter::TimeGetter,
 };
 use crate::{detail::orphan_blocks::OrphanBlocksPool, ChainstateConfig, ChainstateEvent};
 
 type TxRw<'a, S> = <S as Transactional<'a>>::TransactionRw;
 type TxRo<'a, S> = <S as Transactional<'a>>::TransactionRo;
 type ChainstateEventHandler = EventHandler<ChainstateEvent>;
 
-const HEADER_LIMIT: BlockDistance = BlockDistance::new(2000);
+pub const HEADER_LIMIT: BlockDistance = BlockDistance::new(2000);
```

Comment:
It isn't directly related to this pull request, but this constant should probably be moved to the config. Lets add a TODO-node. By the way, the same constant is used in the p2p code...